CONFIG.md

```diff
@@ -132,6 +134,20 @@ Smart routing defines whether the client mode is smart or unisocket. The followi
 
 Default is smart routing mode.
 
+### Enabling Redo Operation
+
+Specifies whether the redo operations are enabled or not.
```

Comment:
Actually we can use Java Client's description of the switch here. It is better
```
     * If true, client will redo the operations that were executing on the server and client lost the connection.
     * This can be because of network, or simply because the member died. However it is not clear whether the
     * application is performed or not. For idempotent operations this is harmless, but for non idempotent ones
     * retrying can cause to undesirable effects. Note that the redo can perform on any member.
```